webcompat/api/endpoints.py

```diff
@@ -91,6 +91,28 @@ def proxy_issues():
     return (issues.content, issues.status_code, get_headers(issues))
 
 
+@api.route('/issues/<username>/<parameter>')
+def get_user_activity_issues(username, parameter):
+    '''API endpoint to return issues related to a user.
+
+    cf. https://developer.github.com/v3/issues/#list-issues-for-a-repository
+    This is only used for "creator" and "mentioned" right now.
+
+    Any logged in user can see details for any other logged in user. We can
+    extend this to non-logged in users in the future if we want.
+    '''
+    if not g.user:
+        abort(401)
+    # copy the params so we can add to the dict.
+    params = request.args.copy()
+    params['state'] = 'all'
```

Comment:
Right now `/issues/hallvors` won't do anything, so it doesn't matter much I think. 

But if we want to allow this kind of fine-grained info in the future we'll need to update this. We're trying to keep the scope of this PR small and make it more interesting in the future. :)